stories/26-Menu.stories.js

```diff
@@ -1,12 +1,12 @@
 import { storiesOf } from '@storybook/vue'
-import { Menu, MenuGroup, MenuButton, MenuList, MenuOptionGroup, MenuItemOption, MenuItem, MenuDivider } from '../packages/kiwi-core/src'
+import { Menu, MenuGroup, MenuButton, MenuList, MenuOptionGroup, MenuItemOption, MenuItem, MenuDivider, Icon } from '../packages/kiwi-core/src'
 
 storiesOf('UI | Menu', module)
   .add('Default Menu', () => ({
-    components: { Menu, MenuGroup, MenuButton, MenuList, MenuItem, MenuDivider },
+    components: { Menu, MenuGroup, MenuButton, MenuList, MenuItem, MenuDivider, Icon },
     template: `
     <Menu>
-      <MenuButton as={Button} rightIcon="chevron-down">
+      <MenuButton as="Button" rightIcon="chevron-down">
```

Comment:
Good catch with this. Was initially using a JSX example in a separate project where I imported these components to test them because I couldn't get them to work in storybook.